admin/src/components/PaperTrail/PaperTrail.jsx

```
@@ -0,0 +1,547 @@
+      
+      if (!window.strapi.paperTrail) {
+        window.strapi.paperTrail = {};
+      }
+      
+      // Expose the function
+      window.strapi.paperTrail.openTrailsModal = manualOpenTrailsModal;
+      
+      return () => {
+        // Clean up when component unmounts
+        if (window.strapi?.paperTrail) {
+          delete window.strapi.paperTrail.openTrailsModal;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)